src/lib/dataset-fetcher/index.ts

```diff
@@ -21,17 +35,38 @@ export type License = {
 export type Dataset = {
   id: string;
   name: string;
-  description?: string;
   publisher: Publisher;
   license: License;
+  description?: string;
+  keywords?: string[];
 };
 
-// TODO: add sorting capabilities
+export enum SortBy {
```

Comment:
Can this be placed outside this file? I cannot import it in client-side code, because of node only imports in this file. Same for SortOrder.

Reply:
Good one! I'll look into this.

Reply:
I will fix this in #57